examples/api/lib/widgets/magnifier/magnifier.0.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatefulWidget {
+  const MyApp({super.key});
+  static const Size loupeSize = Size(200, 200);
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  Offset dragGesturePositon = Offset.zero;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        home: Scaffold(
```

Comment:
Here and elsewhere I see indentation problems where it's 4 instead of 2 spaces.

Reply:
Finally figured this out (tad too late :) ) dart format will do this as a result of not having commas after every closing paren. Added everywhere relevant, should fix both this and the paren on the new line problem.